coderunners/scoring.py

```diff
@@ -0,0 +1,26 @@
+from abc import abstractmethod
+
+from models import Status, TestGroup
+
+
+class AbstractScorer:
```

Comment:
we can have the name `Scorer` to be consistent with other base class names in other. And it should inherit from ABC (from abc). 


Reference: https://github.com/MartinXPN/LambdaJudge/blob/main/coderunners/compilers.py#L10